packages/core/src/components/Modal/WithCross.js

```diff
@@ -31,14 +34,17 @@ class WindowWithCross extends React.Component<Props> {
     variant: 'light',
     classNames: {
       dismissContainer: '',
+      body: '',
+      header: '',
+      footer: '',
     }
   };
 
   render() {
     const { children, onClose, variant, classNames, ...rest } = this.props;
 
     return (
-      <Window {...rest} variant={variant}>
+      <Window {...rest} classNames={classNames} variant={variant}>
```

Comment:
Window isn't expecting the `dismissContainer` as part of the className prop, technically ew should only pass the expected props:
```jsx
const { dismissContainer, ...windowClassNames } = classNames;
return (
  <Window {...rest} classNames={windowClassNames} variant={variant}>
```